packages/charts/src/charts/funnel/funnel-chart.component.test.tsx

```diff
@@ -50,6 +50,17 @@ test('renders correct amount of steps based on provided data', () => {
   expect(steps.length).toEqual(props.data.length);
 });
 
+test('renders values without default formatting', () => {
```

Comment:
As default formatting is not applied I believe the better description will be: `renders values without formatting`